src/drive/locales/fr.json

```diff
@@ -592,6 +592,12 @@
       "cancel": "Annuler",
       "confirm": "J'ai compris"
     },
+    "insideSharedFolder": {
+      "title": "Déplacer vers un dossier partagé ?",
+      "content": "Tous les membres ayant accès à %{destination} auront également accès à %{source}. |||| Tous les membres ayant accès à %{destination} auront également accès aux %{type} sélectionnés.",
```

Comment:
Seems like `type` is always singular? So this would print `accès aux dossier sélectionnés` instead of `accès aux dossierS sélectionnés`?

Reply:
Never mind, this would be fixed by https://github.com/cozy/cozy-drive/pull/3045/files